documentation/src/docs/index.md

```diff
@@ -9,7 +9,7 @@ You may obtain a copy of the License at
 -->
 # Pravega Overview
 
-Pravega is an open source storage primitive implementing **Streams** for continuous and unbounded data. A Pravega stream is a durable, elastic, append-only, unbounded sequence of bytes that has good performance and strong consistency.  
+Pravega is an open source storage primitive implementing **Streams** for continuous and unbounded data. A Pravega stream is a durable, elastic, append-only, unbounded sequence of bytes having good performance and strong consistency.  
```

Comment:
This is wrong. Pravega is not a storage primitive. Pravega is a storage system that exposes stream as a first-class primitive.

There are multiple ways of saying this, but the points that we can't confuse are that:
- Pravega is a storage system
- Stream is the main primitive